core/Pos/Core/Arbitrary.hs

```diff
@@ -71,25 +73,30 @@ deriving instance Random Types.EpochIndex
 
 instance Arbitrary Types.EpochIndex where
     arbitrary = choose (0, maxReasonableEpoch)
+    shrink = genericShrink
 
 instance Arbitrary Types.LocalSlotIndex where
     arbitrary =
         leftToPanic "arbitrary@LocalSlotIndex: " . Types.mkLocalSlotIndex <$>
         choose (Types.getSlotIndex minBound, Types.getSlotIndex maxBound)
+    shrink = genericShrink
 
+-- NOTE: Deriving an 'Arbitrary' instance for wrappers over integral types with 'derive
```

Comment:
Ah, I just now realized I wrote this comment after I used `dumpSplices` to verify that `derive makeArbitrary` does NOT define `shrink` in these case, changed the instances, and then forgot to update the verb's tense.